tests/acceptance/blocking_adapter_test.py

```diff
@@ -205,7 +251,7 @@ def test(self):
 
         # Declare a new queue
         ch.queue_declare(q_name, auto_delete=True)
-        self.addCleanup(self._connect().channel().queue_delete, q_name)
+        self.addCleanup(lambda: self._connect().channel().queue_delete(q_name))
```

Comment:
I changed these to lambda when I discovered that tests that fail with timeout also fail during cleanup - exploding the nosetests log size and creating unnecessary confusion during failure analysis - since the timeout timer in these addCleanup `self._connect()` calls also starts ticking at the same time as the connection used in each test.